docs(config): updates the config sections (#8665)
strimzi/strimzi-kafka-operator: documentation/assemblies/configuring/assembly-scheduling.adoc

// This assembly is included in the following assemblies:
//
// assembly-config.adoc

[id='assembly-scheduling-{context}']
= Configuring pod scheduling

[role="_abstract"]

> To avoid performance degradation caused by resource conflicts between applications scheduled on the same Kubernetes node, it is advisable to schedule Kafka pods separately from critical workloads. 

Reviewer: I would probably avoid the _advisable_ as that would need some more reasoning when, how etc.

```suggestion
To avoid performance degradation caused by resource conflicts between applications scheduled on the same Kubernetes node, you can schedule Kafka pods separately from critical workloads. 
```
